### PR TITLE
lunamedia bidder adapter: change endpoint

### PR DESCRIPTION
**Pull request by lunamedia**

### CODE REVIEW

File: static/bidder-info/lunamedia.yaml [hunk 1]
```diff
@@ -1,18 +1,16 @@
-endpoint: "http://api.lunamedia.io/xp/get?pubid={{.PublisherID}}"
+endpoint: "http://rtb.lunamedia.live/?pid={{.PublisherID}}"
```
Comment (onkarvhanumante) [inline static/bidder-info/lunamedia.yaml:1]: verified endpoint url is reachable

```
~/prebid-repos/prebid-server (tmax/auction-response-fix) » curl -i http://rtb.lunamedia.live                                                    oh0387@15852-p-mac
HTTP/1.1 200 OK
Server: nginx
Date: Wed, 14 Jun 2023 05:08:11 GMT
Transfer-Encoding: chunked
Connection: keep-alive

ok
```

**Review by Sonali-More-Xandr — APPROVED** [review]

File: static/bidder-info/lunamedia.yaml [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
     mediaTypes:
     - banner
     - video
 userSync:
   iframe:
-    url: "https://api.lunamedia.io/xp/user-sync?redirect={{.RedirectURL}}"
-    userMacro: "$UID"
-
+    url: "https://sync.lunamedia.live/psync?t=s&e=0&cb={{.RedirectURL}}"
+    userMacro: "%USER_ID%"
```
Comment (onkarvhanumante) [inline static/bidder-info/lunamedia.yaml:16]: iframe user-sync url is working

**Review by onkarvhanumante — DISMISSED** [review]

**Review by onkarvhanumante — COMMENTED** [review]
@lunamedia , unit test files are still referring to old server url. Requesting to update test to use `http://rtb.lunamedia.live/?pid={{.PublisherID}}`

```
func TestJsonSamples(t *testing.T) {
	bidder, buildErr := Builder(openrtb_ext.BidderLunaMedia, config.Adapter{
		Endpoint: "http://api.lunamedia.io/xp/get?pubid={{.PublisherID}}"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})

	if buildErr != nil {
		t.Fatalf("Builder returned unexpected error %v", buildErr)
	}

	adapterstest.RunJSONBidderTest(t, "lunamediatest", bidder)
}
```

**Comment by lunamedia** [comment]
@onkarvhanumante done

**Review by onkarvhanumante — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**